src/ocean/Assets.ts

```diff
@@ -517,6 +523,11 @@ export class Assets extends Instantiable {
         }
       }
     }
+    if (requiredData && requiredData.userdata)
```

Comment:
we started to like that optional chaining in some other places, so this is the same but shorter:

```js
if (requiredData?.userdata)
```